client/src/components/modal-forms/ManageGraduationForm.js

```diff
@@ -103,6 +107,17 @@ export const ManageGraduationForm = ({ initialValues, onClose, onSubmit, cohortE
                       </MuiAlert>
                     </Box>
                   )}
+                {values?.continue === 'continue_yes' &&
+                  values.status === postHireStatuses.cohortUnsuccessful &&
+                  !roles.includes('health_authority') && (
```

Comment:
I'm not gonna lie, I kind of hate that I'm using AuthContext in this component, but I thought it better than prop drilling a flag